### DISCUSSION TITLE
Gestione ventola Fan SHIM

### DISCUSSION THREAD

**Discussion by viniciogrigoli** [indomus/forum#2329] [HUB: Home Assistant]
Buongiorno a tutti, premesso che sono alla prima esperienza con Home Assistant, volevo gestire la ventola Fan Shim montata sul raspberry pi4; ho seguito l'ottima guida qui presente https://indomus.it/progetti/controllare-il-raffreddamento-di-raspberry-pi-con-home-assistant-e-fan-shim/ ma non capisco questo passaggio : 
"Solitamente si crea una cartella chiamata packages all’interno di quella che contiene tutti i vari file di configurazione YAML; poi, alla voce “homeassistant:” presente nel file si aggiunge una voce analoga alla seguente:
homeassistant:
  packages: !include_dir_named packages"
"homeassistant" dove la trovo, per inserire la striga?
grazie

**Comment by viniciogrigoli** [comment]
forse leggendo attentamente ho capito che dovrebbe essere nel file configuration.yaml?
bene ora funziona, manca solo la parte grafica, vedo le tre entità sotto Impostazioni>Dispositivi e Servizi>entità 
ma come faccio a visualizzarle sulla panoramica? grazie

**Reply by KC-inDomus** [reply]
no, è semplicemente [un package](https://indomus.it/formazione/i-package-di-home-assistant-cosa-sono-e-come-si-installano/)

**Reply by viniciogrigoli** [reply]
> no, è semplicemente [un package](https://indomus.it/formazione/i-package-di-home-assistant-cosa-sono-e-come-si-installano/)

grazie per l'interessamento, ma non ho capito :-)
ho fatto seguendo la guida e quindi creato una cartella packages all'interno della quale ho creato un file fan_shim_home_assistant.yaml editato come segue : 
sensor:
  - platform: command_line
    name: Temperatura CPU
    command: "cat /sys/class/thermal/thermal_zone0/temp"
    unit_of_measurement: "°C"
    value_template: '{{ value | multiply(0.001) | round(1) }}'
    scan_interval: 10
switch:
  - platform: rpi_gpio
    switches:
      - port: 18
        name: "Fan SHIM"
climate:
  - platform: generic_thermostat
    name: Fan SHIM
    heater: switch.fan_shim
    target_sensor: sensor.temperatura_cpu
    min_temp: 50
    max_temp: 70
    ac_mode: true
    target_temp: 50
    cold_tolerance: 0.1
    hot_tolerance: 0.1
    min_cycle_duration:
      seconds: 120
    keep_alive:
      minutes: 5
    initial_hvac_mode: "cool"

poi nel file configuration.yaml ho inserito : 
homeassistant:
  packages: !include_dir_named packages

la ventola funziona e dal menù Impostazioni>Dispositivi e Servizi>Entità vedo le tre entità create e se la apro nelle informazioni vedo i dati, ma nelle impostazioni ho questo messaggio : 

Questa entità ("climate.fan_shim") non ha un ID univoco, pertanto le sue impostazioni non possono essere gestite dall'interfaccia utente

grazie
